extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+lines = sensors_data.readlines()
+
+
+def sensors(lines):
+    #global lines
+    candidate_lines = []
+    for line in lines:
```

Comment:
What happens if someone parses `lines` as a string or dict?
A good idea would be to ensure that whatever is parsed is what you expect.

For example:

```
def capitalize_str(a_str):
    assert isinstance(a_str, str), "Expected a string."
    return a_str.upper()
```

If you parse anything other than a string on the above function, the code will fail, with a message **Expected a string**